Assets/Group1/Scripts/Enemy.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+
+public class Enemy : MonoBehaviour
+{
+    public event UnityAction Dyed;
+
+    [SerializeField] private float _deathRange;
+    [SerializeField] private Player _player;
+
+    private void Update()
+    {
+        if (Vector3.Distance(_player.transform.position, transform.position) < _deathRange)
+        {
+            Die();
+        }
+    }
+
+    private void Die()
+    {
+        Dyed?.Invoke();
```

Comment:
https://trello.com/c/CsOZfMhi/1-%D0%BD%D0%B0%D0%B7%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D0%BE%D0%B2